pkgs/http/lib/src/streamed_response.dart

```diff
@@ -26,3 +26,19 @@ class StreamedResponse extends BaseResponse {
       super.reasonPhrase})
       : stream = toByteStream(stream);
 }
+
+/// This class is private to `package:http` and will be removed when
+/// `package:http` v2 is released.
+class StreamedResponseV2 extends StreamedResponse with BaseResponseWithUrl {
```

Comment:
Don't love the `V2` name, but if it's private and goes away in version 2 of the package, it is probably as direct as it can be. And being ugly is a feature, it makes one want to get rid of it. So LGTM.

Reply:
LOL